### ISSUE TITLE
Protin page is not well designed

### ISSUE THREAD

**Issue by sanjayguwaju** [sanjayguwaju/daily#19]
![image](https://github.com/user-attachments/assets/decce539-005b-4d97-8b0b-5a58d3aa337a)

This is the issue to be fixed

**Comment by aadarshbk** [comment]
can you please show me the relative path for this issue

**Comment by rauniyaraman** [comment]
Yes the codes are inside the file name `**Table of Content/ Protin**`
